[2025-02-02T13:15:04.538+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T13:15:04.551+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.load_data scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-02-02T13:15:04.558+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.load_data scheduled__2025-01-30T00:00:00+00:00 [queued]>
[2025-02-02T13:15:04.558+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-02T13:15:04.568+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2025-01-30 00:00:00+00:00
[2025-02-02T13:15:04.572+0000] {standard_task_runner.py:72} INFO - Started process 155 to run task
[2025-02-02T13:15:04.574+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'load_data', 'scheduled__2025-01-30T00:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpf6b3wl8p']
[2025-02-02T13:15:04.578+0000] {standard_task_runner.py:105} INFO - Job 100: Subtask load_data
[2025-02-02T13:15:04.590+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-02T13:15:04.624+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.load_data scheduled__2025-01-30T00:00:00+00:00 [running]> on host 8dc0a64c5f0f
[2025-02-02T13:15:04.627+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-02T13:15:04.711+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-30T00:00:00+00:00'
[2025-02-02T13:15:04.713+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T13:15:05.061+0000] {base.py:84} INFO - Retrieving connection 'my_dwh_conn'
[2025-02-02T13:15:05.752+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T13:15:05.761+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T13:15:05.761+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=load_data, run_id=scheduled__2025-01-30T00:00:00+00:00, execution_date=20250130T000000, start_date=20250202T131504, end_date=20250202T131505
[2025-02-02T13:15:05.790+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T13:15:05.802+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T13:15:05.812+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
